portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,16 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <div id="title">
+        <h1>Welcome to the Portfolio of Andrés Abundis</h1>
+			</div>
+			<h3 id="text">This is my portfolio! I will present some of my most important aspects.</h3>
+			<div id="image-container">
```

Comment:
Don't forget to update this also to be "selfie-container" :)

Reply:
Okay!